relay-server/src/services/processor/replay.rs

```diff
@@ -175,12 +175,14 @@ fn process_replay_event(
     let mut replay =
         Annotated::<Replay>::from_json_bytes(payload).map_err(ReplayError::CouldNotParse)?;
 
+    if replay.value().is_none() {
+        return Err(ReplayError::NoContent);
+    }
+
     if let Some(replay_value) = replay.value_mut() {
```

Comment:
This should work, `replay_value` goes out of scope before `replay` is used again.

```suggestion
    let Some(replay_value) = replay.value_mut() else {
        return Err(ReplayError::NoContent);
    }
    replay::validate(replay_value)?;
    replay::normalize(&mut replay, client_ip, user_agent);
```